.readthedocs.yml

```diff
@@ -20,7 +20,7 @@ formats: []
 sphinx:
     configuration: docs/rtd-deprecation/conf.py
 python:
-    version: "3.8"
+    version: "3.9"
```

Comment:
This is a marker preventing to merge. Please merge only once Airflow 2.11 cut-off is made.
This PR should NOT go into Airflow 2.10